common/lib/xmodule/xmodule/static_content.py

```diff
@@ -97,7 +97,7 @@ def _write_styles(selector, output_root, classes):
 
     module_styles_lines = [
         "@import 'bourbon/bourbon';",
-        "@import 'base/variables';",
+        "@import 'lms/theme/variables';",
```

Comment:
did the file location move? by chance was base shared across lms/ cms?

Reply:
Yes. I moved this file into a clearer location to make it seem that it is a theme. When these files are compiled for Studio, the LMS Sass directories are on the path, so I believe that it all works.